manager-bundle/src/Resources/skeleton/public/index.php

```diff
@@ -23,6 +23,18 @@
     stream_wrapper_unregister('phar');
 }
 
+// System maintenance mode comes first as it has to work even if the vendor directory does not exist
+if (file_exists(__DIR__.'/../var/maintenance.html')) {
+    $contents = file_get_contents(__DIR__.'/../var/maintenance.html');
+
+    header('HTTP/1.1 503 Service Unavailable', true, 503);
```

Comment:
According to the PHP docs, we should probably use `http_​response_​code()` instead of this.

Reply:
Are you going to fix this in #3883 as well?

Reply:
Not related to the command imho. I'm not entirely sure what the correct headers would be, but it's still on my radar. But feel free to create a PR if you find time, otherwise I'll probably do that next year.